CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.13)
+
+option(FIZZY_TESTING "Enable Fizzy internal tests")
+
+project(fizzy LANGUAGES CXX)
```

Comment:
By default CMake enables both C and C++. Explicitly enabling only C++ makes configuration slightly faster.